File: sdk-examples/java/AwsDynamoDbExample.java
// Example: Inserting an item into DynamoDB using AWS SDK for Java v2

import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
import software.amazon.awssdk.services.dynamodb.model.PutItemRequest;
import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
import java.util.HashMap;

public class AwsDynamoDbExample {
    public static void main(String[] args) {
        DynamoDbClient dynamoDb = DynamoDbClient.create();

        HashMap<String, AttributeValue> itemValues = new HashMap<>();
        itemValues.put("Id", AttributeValue.builder().s("101").build());
        itemValues.put("Name", AttributeValue.builder().s("Sample Item").build());

        PutItemRequest request = PutItemRequest.builder()
                .tableName("YourTableName") // Replace with your table name
                .item(itemValues)
                .build();

        dynamoDb.putItem(request);

        System.out.println("Item inserted into DynamoDB.");

        dynamoDb.close();
    }
}

File: sdk-examples/dotnet/AwsDynamoDbExample.cs
// Example: Inserting an item into DynamoDB using AWS SDK for .NET

using Amazon.DynamoDBv2;
using Amazon.DynamoDBv2.Model;
using System;
using System.Collections.Generic;
using System.Threading.Tasks;

class AwsDynamoDbExample
{
    static async Task Main()
    {
        var dynamoDbClient = new AmazonDynamoDBClient();

        var item = new Dictionary<string, AttributeValue>
        {
            { "Id", new AttributeValue { S = "101" } },
            { "Name", new AttributeValue { S = "Sample Item" } }
        };

        var putRequest = new PutItemRequest
        {
            TableName = "YourTableName", // Replace with your table name
            Item = item
        };

        var response = await dynamoDbClient.PutItemAsync(putRequest);
        Console.WriteLine("Item inserted into DynamoDB.");
    }
}